recipes/ecodata/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "ecodata" %}
+{% set version = "0.2.2rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jemissik/ecodata/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: aeec406e855285e24b03cbce0b22ab5d406299576abfeb74bcc784dd289d80d4
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
```

Should be the same as run.

```suggestion
    - python >3.8
```